v2/tests/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'aruneli'
```

Comment:
We've avoided adding the __author__  header to python files. Can you remove? There is also a setting in pycharm to control the template for new python files.